### ISSUE TITLE
Can we get a display type text

### ISSUE THREAD

**Issue by Madcliff** [Luzifer/streamdeck#5]
So, I use my streamdeck to display unicode characters for writing Biblical Greek. I would appreciate being able to just specify the button to have a string as its image rather than needing to find a useless command

**Comment by Luzifer** [comment]
You can just use a simple `echo` to display text:

```yaml
      3:
        display:
          type: exec
          attributes:
            command: [/usr/bin/bash, -c, "echo -n 'AFK'"]
        actions: []
```

I'm not sure there is any value in having a separate display element doing the same logic which is already there…

**Comment by Luzifer** [comment]
Changed my mind and implemented this display element: 728ec0a62913946fcd47082a227595db5f8e1820

**Comment by Luzifer** [comment]
The new element is supported in `v1.3.0` and is now documented in the [Wiki](https://github.com/Luzifer/streamdeck/wiki#text)
